### ISSUE TITLE
RESTful API

### ISSUE THREAD

**Issue by benbalter** [18F/jekyll_pages_api#14]
Rather than a single firehose of a site's entire content (which will get big fast), it'd be awesome if I could simply add `.json` to any Jekyll URL to get the JSON representation of just that file, essentially turning any Jekyll site into a RESTful API.

**Comment by afeld** [comment]
Whoa, that's a *great* idea. Do you mean adding that functionality to this gem, or to Jekyll itself?

**Comment by benbalter** [comment]
>  Do you mean adding that functionality to this gem, or to Jekyll itself?

Yes. Assumed it'd be easier via the gem (if you already have the Jekyll -> JSON logic), it'd just be a matter of adding a generator that spits out a JSON file per file.

**Comment by tomjohnson1492** [comment]
I agree. This addition would be awesome.

**Comment by deedeelavinder** [comment]
I can take a look at this one.
